CenterForOpenScience/osf.io
# Remove usages of website utilities outside of the website directory [OSF-9001]

## Purpose

This is in anticipation of a much larger refactor where the website directory is removed entirely. 

## Changes

Remove website utils that are being used outside of the website directory to a more central place that is using them

## QA Notes



This should not change any user-facing things, but there is a lot of potential for error as this touches so much code. This and #8095 are both essentially "move a bunch of things around and make sure the new places are referenced correctly." It'd be good to test this and #8095 at the same time, as they're very similar in scope.

## Side Effects

none anticipated, though this touches a LOT of imports, so nervous about that...

## Ticket
https://openscience.atlassian.net/browse/OSF-9001
